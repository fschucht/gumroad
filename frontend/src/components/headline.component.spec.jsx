import renderer from 'react-test-renderer';
import { Headline } from './headline.component';

describe('Headline', () => {
  describe('when size is large', () => {
    it('should render a Headline component', () => {
      const renderedComponent = renderer.create(<Headline size='large'>Hello World!</Headline>).toJSON()

      expect(renderedComponent).toMatchSnapshot()
    })
  })

  describe('when size is missing', () => {
    it('should return null', () => {
      const renderedComponent = renderer.create(<Headline>Hello World!</Headline>).toJSON()

      expect(renderedComponent).toMatchSnapshot()
    })
  })
})