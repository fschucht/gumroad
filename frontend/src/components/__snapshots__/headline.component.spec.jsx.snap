// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline when size is large should render a Headline component 1`] = `
<h1
  className="headline--large"
>
  Hello World!
</h1>
`;

exports[`Headline when size is missing should return null 1`] = `null`;
